installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -81,11 +82,13 @@ defmodule <%= @web_namespace %> do
 
   defp html_helpers do
     quote do
+      # Translation
```

Comment:
This needs to be inside the `if @gettext`. Those new lines are tricky to get right, my suggestion is to do as few changes as possible. :)

Reply:
Also fixed and tried with `--gettext` and `--no-gettext`.